manipulation/station.py

```diff
@@ -615,78 +584,28 @@ def _ApplyDriverConfigSim(
             controller_iiwa = AddPlanarIiwa(controller_plant)
         else:
             controller_iiwa = AddIiwa(controller_plant)
+        # TODO: Add the model that is welded to the iiwa link 7 rather than always
+        # adding the WSG.
         AddWsg(controller_plant, controller_iiwa, welded=True)
         controller_plant.Finalize()
```

Comment:
We must keep `controller_plant` alive per contract with `AddSimIiwaDriver`. Otherwise, this will result in a segmentation fault. I don't think there is a neat `keep_alive` solution on the C++ side here (`InverseDynamicsController` secretly uses a `keep_alive` trick and thus does not rely on its contract). I will think of a neat way of doing this.

Reply:
the pointer system takes care of this :tada: